### ISSUE TITLE
Dev: Post Mortem: Unable to log into admin or security accounts

### ISSUE THREAD

**Issue by Tony-Villa** [hackforla/tdm-calculator#2617]
### Overview
This has already been done, this is just a paper trail so we have something to point back to if there are any ripples from this change.

#### Problem
- At 2:34 PM, Bonnie tagged Tony(me) on a slack message saying she couldn't log into the Admin or Security accounts.
- At approx. 5:30 PM, I tried and failed to log into those 2 accounts. As Bonnie mentioned, the regular users are still working as expected. I logged the object coming back from the database and noticed that the `emailConfirmed` property for those accounts was coming back as false causing the issue.
- At approx. 6:30 PM -- during the dev stand up, Ahn mentioned that she used the app to change the name on the account in order test truncating on cells. 
    - Current theory is that changing the name on an account causes the `emailConfirmation` to swap from true to false without sending a verification email. (Still needs to be confirmed)
- At approx. 7:30 PM -- during the all hands meeting, Bonnie tried to reset the password to get an email confirmation but it never came (which might be another bug). 

#### Solution
- After confirming that the `emailConfirmation` being false was in fact the reason we couldn't log in, I wrote a sql query to change the `emailConfirmation` column on 2 rows in the `Login` table. The two rows were the ones containing the Admin and the Security users.
- Once I changed them back from `0` to `1`, we were able to log back into those accounts.

**Comment by Tony-Villa** [comment]
@entrotech pinging for visibility
